### PR TITLE
Wip/mkow/python nits

### PR DESCRIPTION
**Pull request by mkow**
low priority nits.

### CODE REVIEW

File: python/test_dis [hunk 1]
```diff
@@ -14,15 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from veles.dis.isa.falcon import FalconIsa
-from veles.data.bindata import BinData
 import sys
+from pprint import pprint
+from veles.data.bindata import BinData
```
Comment (altran01) [inline python/test_dis]: If you are so strict about fixing nits include an empty line between stdlib and local imports :)

File: python/setup.cfg [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [metadata]
 name = veles
-home-page = https://codisec.com/veles/
+home-page = https://veles.io/
 author = CodiSec
-author-email = veles@codisec.com
+author-email = contact@veles.io
```
Comment (adamiwaniuk) [inline python/setup.cfg]: please update email also in other places

File: python/test_dis (hunk 1 above)
Comment (altran01) [inline python/test_dis]: Also why is this only file where you moved imports around?

File: python/sub [hunk 3]
```diff
@@ -4,11 +4,9 @@ import sys
...
+from pprint import pprint
 
 client = client.create_client(sys.argv[1], sys.argv[2])
-
 obj = nodeid.NodeID.from_hex(sys.argv[3])
-
-import pprint
 for x in client.get_sub(obj):
-	pprint.pprint(x)
+	pprint(x)
```
Comment (altran01) [inline python/sub]: When you are touching lines with tabulation can you change it to spaces as well :) ?

Comment (altran01) [inline python/sub]: `python/listsub` has construct identical to what you changed here - any reason you didn't do it there?

File: python/show [hunk 4]
```diff
@@ -4,10 +4,8 @@ import sys
 
 from veles.schema import nodeid
 from veles.scli import client
+from pprint import pprint
```
Comment (altran01) [inline python/show]: Nit. order imports according to PEP8.

File: python/test_dis (hunk 1 above)
Comment (mkow) [inline python/test_dis]: Will fix in next PR.

**Pull request merged**